src/main/java/org/broadinstitute/hellbender/tools/walkers/CombineGVCFs.java

```diff
@@ -157,8 +157,14 @@ void createIntermediateVariants(SimpleInterval intervalToClose) {
 
         // Break up the GVCF according to the provided reference blocking scheme
         if ( multipleAtWhichToBreakBands > 0) {
-            for (int i = ((intervalToClose.getStart())/multipleAtWhichToBreakBands)*multipleAtWhichToBreakBands; i <= intervalToClose.getEnd(); i+=multipleAtWhichToBreakBands) {
-                sitesToStop.add(i-1); // Subtract 1 here because we want to split before this base
+            // if the intermediate interval to close starts before the end of the first interval,
```

Comment:
This seems like a perfectly reasonable fix but i would probably add an extra layer of protection here. Below where sitesToStop is converted into intervalLocs you should also add a check that the intervals generated by this or the other break code are not going to be invalid. Also i think this is still broken with multipleAtWhichToBreakBands = 1 because you get a start position = 0

Reply:
@jamesemery I'm not clear on what kind of interval validation you're suggesting - if we ever do create invalid interval I think it would fail the same way it does now. You're right about the start position 0 - I wound up factoring out the banded stop site calculation for bands into a separate method that can be unit tested, and added a bunch of test cases that verify the stop sites that are generated, and that they result in valid intervals.